### DISCUSSION TITLE
Add seperate itemframe perms for each CMD arg

### DISCUSSION THREAD

**Discussion by AZFire1** [Zrips/CMI#5754] [Enhancement]
Like many other commands, could you add a seperate required perm for each CMD arg of /itemframe? E.g. "cmi.command.itemframe.invisible/fixed/invulnerable"

A few examples of various commands with perms already for each arg include: homes, donate, mail, etc.

This would be super useful, as I'd like the "/itemframe invisible/fixed" to be a donator perk on my server.

**Comment by gecko10000** [comment]
You could accomplish this with specialized commands' `bperm` condition, and a custom alias.
